client/src/components/Room/List.js

```diff
@@ -0,0 +1,70 @@
+import Card from "./Card";
+
+function List(props) {
+  const seatedPlayers = props.seatedPlayers;
+  const length = seatedPlayers && seatedPlayers.length;
+
+  function computeAverage() {
+    const playedCards = seatedPlayers
+      // Get the card values.
+      .map((player) => {
+        if (player.card === "½") {
+          return "0.5"; // For easier number conversion.
+        }
+        return player.card;
+      })
+      // Excludes the players that did not vote or voted on "?".
+      .filter((card) => {
+        switch (card) {
+          case "":
+          case "?":
+            return false;
+
+          default:
+            return true;
+        }
+      })
+      .map((string) => Number(string));
+    return playedCards.reduce((a, b) => a + b, 0) / playedCards.length;
```

Comment:
No need to map filter and map and then reduce on this case. Try to use only the reduce method with an initial value. 
TIP: 
intial object = {total: 0, validVotes: 0}